src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_InternalsVisibleTo.vb

```diff
@@ -639,5 +639,265 @@ using System.Reflection;
                 Await state.AssertNoCompletionSession()
             End Using
         End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        <WorkItem(29447, "https://github.com/dotnet/roslyn/pull/29447")>
+        Public Async Function CodeCompletionReplacesExisitingAssemblyNameWithDots_1() As Task
```

Comment:
@jasonmalinowski This method should use the same approach as the existing test located above it. Same with the other methods which appear added in this file.

Reply:
Fixing, although @sharwell and @ivanbasov I'm confused. The state here where some things are using Theories and some are using Facts is the current state in master-vs-deps, and not something being brought by this merge. Do we have other tests still broken?

Reply:
Good catch! It seems that those WpfFacts were added there after I migrated tests to WpfTheories. It seems I have to migrate those new tests too :-(